ci/jenkins/jobs/macros.yaml

```diff
@@ -11,15 +11,34 @@
     builders:
       - shell: |-
           #!/bin/bash
-          set -ex
+
+          saveLogs() {
+            echo "=== Remove too many logs ==="
```

Comment:
Maybe change the message to "Truncate old logs"

Reply:
Updated.